src/main/java/com/recruitment/project/parking/parkingspaces/domain/UserParkingVisit.java

```diff
@@ -0,0 +1,26 @@
+package com.recruitment.project.parking.parkingspaces.domain;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.Data;
+import net.minidev.json.annotate.JsonIgnore;
+
+import javax.persistence.*;
```

Comment:
takie importy utrudniają review, bo nie wiadomo skąd jest dana klasa

Reply:
Czy chodzi tutaj o to, że pakiet javax.persistence jest zwinięty, każdy import powinien być w oddzielniej linni, tak? Bo tutaj korzystałem ze skróty IneliJ do imporu i po 5 imporcie z  javax.persistence zawinął w *.